src/test/java/guitests/UpdateTaskCommandTest.java

```diff
@@ -140,6 +140,160 @@ public void updateEvents() {
     }
 
```

Comment:
Summary of tests below this line.....
Event -> Task with deadline
Event -> Floating
Task with deadline -> Event
Task with deadline -> Floating
Floating -> Event
Floating -> Task with deadline
All of which should be updated correctly

Event -> Floating + Task with deadline
Event -> Event + Floating Task
Event -> Event + Task with deadline
Event -> All floating, task with deadline, and event
All of these should fail to update and the original event should still be around.